plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -402,4 +402,16 @@ String columnSchema()
             return "%s %s NOT NULL%s".formatted(name, type, propertiesSchema.isEmpty() ? "" : " WITH (%s)".formatted(propertiesSchema));
         }
     }
+
+    @Test
+    void testRenameTable()
+    {
+        assertUpdate("CREATE TABLE faker.default.original_table (id INTEGER, name VARCHAR)");
```

Comment:
`faker.default` is the default catalog and schema name in this test class. No need to repeat that. Please remove it.